clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as

```diff
@@ -1,28 +1,29 @@
 package org.bigbluebutton.lib.user.services {
 	import mx.utils.ObjectUtil;
-	
```

Comment:
This file was edited to remove the tabs on empty lines, but the FlexFormatter definition includes them.